src/watchmaker/config/status/__init__.py

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8 -*-
+"""Status Config module."""
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals, with_statement)
+
+import logging
+
+STATUS = {
+    "RUNNING": "Running",
+    "COMPLETE": "Completed",
+    "ERROR": "Error",
+}
+
+
+def is_valid(config):
+    """Validate config."""
+    if not config:
+        return True
+
+    targets = config.get("targets", None)
+    if not targets:
+        return False
+
+    valid = True
+    for target in targets:
+        if "key" not in target or not target["key"]:
+            valid = False
+            logging.error("Status target is missing key or value")
+        if "target_type" not in target or not target["target_type"]:
+            valid = False
+            logging.error("Status target is missing target_type or value")
+        if "required" in target:
```

Comment:
Can collapse the two if statements using get:

```suggestion
        if not isinstance(target.get("required"), bool):
```